content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -348,6 +351,33 @@ fill out the :guilabel:`Cancel reason` and :guilabel:`Cancel remarks` fields. Th
      the government portal. You can click :guilabel:`Process Now` if you want to process the invoice
      immediately.
 
+.. _india/gstin_status:
+
+Indian Check GSTIN Status
+=========================
+
+The :guilabel:`Indian - Check GST Number Status` module allows you to verify the status of a GSTIN
+(Goods and Services Tax Identification Number) directly from Odoo.
+
+There are two ways to verify the status of a GST Number in Odoo:
```

Comment:
I would remove this sentence: the "two ways" in question are not really related to one another since they do not concern the same thing (one is for contacts and the other for invoices/bills).